### PR TITLE
kcat: add Dutch translation

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]

File: pages.nl/common/kcat.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# kcat
+
+> Apache Kafka produceer en consumeer tool.
+> Meer informatie: <https://github.com/edenhill/kcat>.
+
+- Consummeer berichten startend met de nieuwste offset:
```
Comment (leonvsc) [inline pages.nl/common/kcat.md:6]:
```suggestion
- Consumeer berichten startend met de nieuwste offset:
```

File: pages.nl/common/kcat.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`kcat -C -t {{onderwerp}} -b {{makelaars}}`
+
+- Consummeer berichten startend met de oudste offset en sluit af nadat het laatste bericht is ontvangen:
```
Comment (leonvsc) [inline pages.nl/common/kcat.md:10]:
```suggestion
- Consumeer berichten startend met de oudste offset en sluit af nadat het laatste bericht is ontvangen:
```

File: pages.nl/common/kcat.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+`kcat -C -t {{onderwerp}} -b {{makelaars}} -o beginning -e`
+
+- Consummeer berichten als een Kafka consumeer groep:
```
Comment (leonvsc) [inline pages.nl/common/kcat.md:14]:
```suggestion
- Consumeer berichten als een Kafka consumeer groep:
```

**Review by leonvsc — APPROVED** [review]
LGTM after the suggestions!

**Pull request merged**